sal_interfaces/WeatherForecast/WeatherForecast_Telemetry.xml

```diff
@@ -26,649 +26,649 @@
-      <Units>deg</Units>
-      <EFDB_Name>temperature_spread</EFDB_Name>
+      <Description>temperature spread</Description>
+      <Units>deg_C</Units>
+      <EFDB_Name>temperatureSpread</EFDB_Name>
     </item>
     <item>
       <IDL_Type>float</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast precipitation</Description>
+      <Description>precipitation</Description>
       <Units>millimeter</Units>
       <EFDB_Name>precipitation</EFDB_Name>
     </item>
     <item>
       <IDL_Type>long</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast precipitation probability</Description>
+      <Description>precipitation probability</Description>
       <Units>millimeter</Units>
-      <EFDB_Name>precipitation_probability</EFDB_Name>
+      <EFDB_Name>precipitationProbability</EFDB_Name>
     </item>
     <item>
       <IDL_Type>float</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast precipitation spread</Description>
+      <Description>precipitation spread</Description>
       <Units>millimeter</Units>
-      <EFDB_Name>precipitation_spread</EFDB_Name>
+      <EFDB_Name>precipitationSpread</EFDB_Name>
     </item>
     <item>
       <IDL_Type>float</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast maximum windspeed</Description>
+      <Description>maximum windspeed</Description>
       <Units>meter/second</Units>
-      <EFDB_Name>windspeed_max</EFDB_Name>
+      <EFDB_Name>windspeedMax</EFDB_Name>
     </item>
     <item>
       <IDL_Type>float</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast minimum wind speed</Description>
+      <Description>minimum wind speed</Description>
       <Units>meter/second</Units>
-      <EFDB_Name>windspeed_min</EFDB_Name>
+      <EFDB_Name>windspeedMin</EFDB_Name>
     </item>
     <item>
       <IDL_Type>float</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast mean windspeed</Description>
+      <Description>mean windspeed</Description>
       <Units>meter/second</Units>
-      <EFDB_Name>windspeed_mean</EFDB_Name>
+      <EFDB_Name>windspeedMean</EFDB_Name>
     </item>
     <item>
       <IDL_Type>float</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast windspeed spread</Description>
+      <Description>windspeed spread</Description>
       <Units>meter/second</Units>
-      <EFDB_Name>windspeed_spread</EFDB_Name>
+      <EFDB_Name>windspeedSpread</EFDB_Name>
     </item>
     <item>
       <IDL_Type>long</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast wind direction</Description>
+      <Description>wind direction</Description>
       <Units>deg</Units>
-      <EFDB_Name>wind_direction</EFDB_Name>
+      <EFDB_Name>windDirection</EFDB_Name>
     </item>
     <item>
       <IDL_Type>long</IDL_Type>
       <Count>14</Count>
-      <Description>Forecast maximum sea level pressure</Description>
+      <Description>maximum sea level pressure</Description>
       <Units>bar</Units>
```

Comment:
`bar` is not an SI unit. Please change to Pascal here and everywhere else. Sorry, I missed this during the first pass.